### PR TITLE
chore: Migrate UpdateCopyrightPresenter to ViewModel

### LINKED ISSUE #1546: Migrate UpdateCopyrightPresenter to ViewModel
**Describe the chore**
- Migrate UpdateCopyrightPresenter to ViewModel
- Write tests for UpdateCopyrightViewModel

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

Parent issue:
#562

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1546 

Checklist:

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 

Migrated UpdateCopyrightPresenter to ViewModel

### CODE REVIEW

**Comment by ShridharGoel** [comment]
> @ShridharGoel Please also add the tests.

Okay

**Comment by ShridharGoel** [comment]
@Masquerade0097 I'm experiencing some problem with the tests in this case. Can you please have a look?

File: app/src/main/java/com/eventyay/organizer/core/event/copyright/update/UpdateCopyrightViewModel.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import com.eventyay.organizer.utils.StringUtils;
+
+import javax.inject.Inject;
+
+import io.reactivex.disposables.CompositeDisposable;
+
+public class UpdateCopyrightViewModel extends ViewModel {
+
+    private final CopyrightRepository copyrightRepository;
+    private Copyright copyright;
```
Comment (Masquerade0097) [inline app/src/main/java/com/eventyay/organizer/core/event/copyright/update/UpdateCopyrightViewModel.java]: Initialize it.

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/event/copyright/update/UpdateCopyrightViewModel.java]: Fixed. Thanks. :)

**Pull request merged**